test/SmokeTests/OOProcSmokeTests/durableJS/DurableFunctionsOrchestratorJS/index.js

```diff
@@ -13,12 +13,15 @@ const df = require("durable-functions");
 
 module.exports = df.orchestrator(function* (context) {
     const outputs = [];
+    const city = {city:"Paris", country:"France"};
 
     // Replace "Hello" with the name of your Durable Activity Function.
     outputs.push(yield context.df.callActivity("Hello", "Tokyo"));
     outputs.push(yield context.df.callActivity("Hello", "Seattle"));
     outputs.push(yield context.df.callActivity("Hello", "London"));
+    outputs.push(yield context.df.callActivity("Hello", "123"));
```

Comment:
nit: can we add a case with numbers as input?

Reply:
This is testing a string with numbers, which is a good edgfe case, but one with "real" ints would be good

Reply:
Yes, I didn't mean to make it a string. I updated it to `123`.